back/app/models/custom_form.rb

```diff
@@ -52,4 +55,29 @@ def heal_fields_ordering!
   def fields_updated!
     update!(fields_last_updated_at: Time.now)
   end
+
+  private
+
+  def sanitize_print_start_multiloc
+    return unless print_start_multiloc&.any?
+
+    self.print_start_multiloc = sanitize_multiloc(print_start_multiloc)
+  end
+
+  def sanitize_print_end_multiloc
+    return unless print_end_multiloc&.any?
+
+    self.print_end_multiloc = sanitize_multiloc(print_end_multiloc)
+  end
+
+  def sanitize_multiloc(multiloc)
+    service = SanitizationService.new
+
+    multiloc = service.sanitize_multiloc(
+      multiloc,
+      %i[title alignment list decoration image video]
```

Comment:
@jinjagit Code looks good, but just to confirm. Does this cover all the options in this pallette? You can remove `video` though. Videos don't work so well when printed :-)

<img width="487" alt="image" src="https://github.com/user-attachments/assets/18612eb8-aa7a-46c7-8b4b-3fd269335dbb" />

 following

Reply:
I think it does, but I'll check. Will remove video. Thanks.